test/fixtures/experimental/import-meta/without-dynamic-import/actual.js

```diff
@@ -0,0 +1,4 @@
+const x = import.meta;
+const url = import.meta.url;
+import.meta;
+import.meta.url;
```

Comment:
No deeper reason than "order of copy&paste". ^^ Copied the `actual.js`/`expected.json` over again to keep them in sync.